src/storage/src/hummock/store/version.rs

```diff
@@ -1211,6 +1213,29 @@ impl HummockVersionReader {
                 }
                 Ok(builder.finish())
             }
+            VectorIndexImpl::HnswFlat(hnsw_flat) => {
+                let Some(graph_file) = &hnsw_flat.graph_file else {
+                    return Ok(vec![]);
+                };
+
+                let graph = self.sstable_store.get_hnsw_graph(graph_file).await?;
+
+                // TODO: avoid vector_files deep-copy
+                let vector_store = FileVectorStore::new(
```

Comment:
`FileVectorStore` is has nothing to do with hnsw. Should we use `FileVectorStore` for `VectorIndeImpl::Flat` as well?